docker/nvidia/Dockerfile_nvidia

```diff
@@ -1,6 +1,6 @@
-FROM dalg24/dtk-stack:latest
+FROM @BASE_IMAGE@
```

Comment:
I don't see it. I see that `placeholder = @BASE_IMAGE@` but I don't see where/how you set `@BASE_IMAGE@` itself.